### ISSUE TITLE
[8] Values in repeated metadata fields not displaying in intended order

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/palni-palci#119]
Display in order that they were entered. Should this be newest to oldest or vice versa?

What fields is this important? All of them?

Is the display consistently opposite or is the display random?

**Comment by DiemBTran** [comment]
Amanda - Sept 2020:

---

user feedback: "In the Title field I put the journal title and the year. I’ve added a second title line so that I can put the Volume number. The system keeps flipping it so that volume is the primary title. This isn’t the only field where that’s happening. I’ve also noticed it in the abstract/summary field. It’s not an end of the world kind of thing, but it is a little annoying." https://hanover.hykucommons.org/collections/ec4beba9-7d19-41c4-aa7a-2d1307865f9c?locale=en

testing from Nic" I tested this in all the metadata fields for the four worktypes, and this occurs consistently in all fields, including dropdown lists. I created an example work in the sandbox that makes the behavior really clear. If you edit the work and save without making any changes, it will flip the order of the two term fields. It will do this repeatedly as you edit and save, and also does this upon the initial creation of a work. I also created an example work in production that behaves slightly differently. It will flip the terms upon initial creation of the work, and if you add a third term it will bump it to the top of the list, but doesn't do the consistent flipping without changes when editing and re-saving the work.
https://nic.commons-archive.org/concern/generic_works/45532335-5217-424a-a34f-21102ee87297?locale=en
